script.js

```diff
@@ -1,136 +1,451 @@
+    ],
+    heading: [
+      'Multi Post Stories',
+      'Keeping track of hundreds of components website',
+    ],
+    technologies: ['html', 'css', 'Javascript'],
+    image: './images/popDesktop.png',
+    alt: ['ten', 'icon'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    btns: ['See live', 'See source'],
+    image2: './images/Icon.png',
+  },
+];
+
+const portfolio = document.getElementById('portfolio');
+
+worksInfo.map((elem) =>
+  // eslint-disable-next-line implicit-arrow-linebreak
+  elem.classes.map((e) => {
+    const div = document.createElement('div');
+    div.className = e;
+    return portfolio.appendChild(div);
+  }));
+const plchdr = document.querySelector('.plchdr');
+const img = document.querySelector('.img');
+const text = document.querySelector('.text');
+const image = document.createElement('img');
+const card = document.querySelectorAll('.card');
+const blur = document.querySelector('#portfolio');
+const openPopUp = document.querySelector('.popup');
+const h2 = document.createElement('h2');
+const para = document.createElement('p');
+const ul = document.createElement('ul');
+const btn = document.createElement('button');
+worksInfo.map((elem) => {
+  elem.heading.map((e) => {
+    if (e === 'Professional Art Printing Data') {
+      for (let i = 0; i < card.length; i += 1) {
+        const cardArr = card[i];
+        const h3 = document.createElement('h3');
+        h3.innerHTML = e;
+        cardArr.appendChild(h3);
+      }
+    }
+  });
+  for (let i = 0; i < card.length; i += 1) {
+    const para = document.createElement('p');
+    const cardArr = card[i];
+    para.innerHTML = elem.description;
+    cardArr.appendChild(para);
+  }
+  for (let i = 0; i < card.length; i += 1) {
+    const ul = document.createElement('ul');
+    const cardArr = card[i];
+    elem.technologies.map((tech) => {
+      const li = document.createElement('li');
+      const aTag = document.createElement('a');
+      aTag.innerHTML = tech;
+      li.appendChild(aTag);
+      ul.appendChild(li);
+      return cardArr.appendChild(ul);
+    });
+  }
+  for (let i = 0; i < card.length; i += 1) {
+    const btn = document.createElement('button');
+    const cardArr = card[i];
+    btn.innerHTML = elem.btn;
+    worksInfo.map((e) => {
+      e.classes.map((item) => {
+        if (item === 'seeDemo2') {
+          const seeDemo = btn;
+          seeDemo.addEventListener('click', () => {
+            blur.classList.add('active');
+            openPopUp.classList.add('active');
+          });
+          btn.className = item;
+        }
+      });
+      return cardArr.appendChild(btn);
+    });
+  }
+});
```

Comment:
- [ ] Kindly, as requested by the previous reviewer 🔍, the buttons to open the modal window in the desktop version are not appearing for the rest of the projects. Please add them 🎲.

![image](https://user-images.githubusercontent.com/91301423/204093572-b6f00c81-cc1c-45a4-b61e-e57b165f3ef8.png)

Reply:
The buttons should only appear in the mobile version.
They are hidden in the Desktop version